### DISCUSSION TITLE
Use micro frontends (e.g. single-spa or qiankun) as an alternative to iframes for integration with other applications

### DISCUSSION THREAD

**Discussion by TaitChan** [grafana/grafana#60720] [General]
Use micro frontends (e.g. single-spa or qiankun) as an alternative to iframes for integration with other applications

Why Micro frontends？

Iframes are fine in most cases, but there are some cases where I don't get a better experience.
For example, popovers and drawers are displayed only within the viewable area of the iframe container.
When I use qiankun, the display area of the drawer is global, 
which eliminates the fragmentation that occurs when the page is integrated.

——————————————————————————————————————————————————————

使用微前端（例如 single-spa 或者是 qiankun ）作为不同于 iframe 的其它方案达到被其它应用集成的效果

为什么使用微前端？

iframe 在多数场景下都是正常选择，但是在有些情况下我不能获得更好的体验，
比如弹窗和抽屉，它们仅展示在 iframe 容器的可视区域内，
而当我使用 qiankun 的时候，抽屉的显示区域是全局，
这样可以消除当页面被集成的时候产生的割裂感。

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
